ote/resources/public/language/sv.edn

```diff
@@ -424,6 +424,11 @@
  :organization-page
  {:contact-types            "Yhteystavat"
   :organization-form-title  "Uppassningenstuottajan tiedot"
+  :help-desc-1              "Tiedot tulevat näkyviin NAP-palvelun julkisille sivuille."
```

Comment:
You could just use single :help-desc that is a vector of parts and parameterized.